src/shared/ui/search/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { type FormEvent, type FC } from 'react';
+import { Icon } from '@shared/icons/Icon';
+
+export const SearchComponent: FC = () => {
```

Comment:
```suggestion
export const SearchInput: FC = () => {
```

Also folder should be named as component